firefox-ios/Client/Frontend/Autofill/Address/AddressAutofillToggle.swift

```diff
@@ -0,0 +1,106 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import SwiftUI
+import Shared
+
+// MARK: - AddressAutofillToggle
+
+/// A SwiftUI view representing a toggle with theming for address autofill functionality.
+struct AddressAutofillToggle: View {
```

Comment:
`CreditCardAutofillToggle` and `AddressAutofillToggle` seem to be very similar, what about creating a generic Toggle that both can use and we pass in the necessary data? 

Therefore, we can have some consistency in the cells. It seems on iPad, the two differ a bit (toggle is far vs. close to the text).
| Credit Card | Address|
| --- | --- |
|![simulator_screenshot_F07C47A4-8D73-4C2C-A247-633242366CB7](https://github.com/mozilla-mobile/firefox-ios/assets/6743397/b76193b6-51e2-4609-a8e1-6d843a20eb9e) | ![simulator_screenshot_17D603B9-640B-4537-A336-597D432A7AD8](https://github.com/mozilla-mobile/firefox-ios/assets/6743397/128f7c18-2fa6-4519-b9b0-371f803048bf)|

Reply:
the one on the left is the correct one with theme, generic is better but even if we don't want to make generic one for this PR we should still match the theme for both of them @jnrahme

Reply:
iPad Air (5th gen) - iOS 16.6

Reply:
Resolved thank you good catch !

Reply:
<img width="661" alt="Screenshot 2024-01-11 at 5 03 12 PM" src="https://github.com/mozilla-mobile/firefox-ios/assets/11262696/8386561f-ee3e-4acf-b889-70b02d63b49f">
<img width="668" alt="Screenshot 2024-01-11 at 5 03 26 PM" src="https://github.com/mozilla-mobile/firefox-ios/assets/11262696/f9282671-d79f-40ca-aed8-7c21ccb08385">

Reply:
great, thank you!